src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,83 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+from argparse import ArgumentParser
+
+import pandas as pd
+
+from utils import remove_angle_brackets
+
+
+SUBSET_MAP = {
+    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disease',  # disorder
+    'Orphanet:557492': 'http://purl.obolibrary.org/obo/mondo#ordo_group_of_disorders',  # group of disorders
+    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_clinical_subtype',  # subtype of a disorder
+}
+ROBOT_TEMPLATE_HEADER = {
+    'mondo_id': 'ID',
+    'ordo_id': 'A oboInOwl:hasDbXref',
+    'subset': 'A oboInOwl:inSubset',
+}
+
+
+def create_ordo_subsets_robot_template(
+    class_subsets_tsv_path: str, mondo_mappings_path: str, outpath: str,
+):
+    """Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+    # Read: mondo.sssom.tsv
+    mondo_df = pd.read_csv(mondo_mappings_path, sep='\t', comment='#').rename(columns={
+        'subject_id': 'mondo_id',
+        'subject_label': 'mondo_label',
+    })
+    # - keep only exact matches
+    mondo_df = mondo_df[mondo_df['predicate_id'] == 'skos:exactMatch']
```

Comment:
I'm not sure if the list of Mondo IDs to tag with RD tags should be filtered here.

Reply:
You mean the rows of `mondo.sssom.tsv` should not be filtered here? Perhaps. FYI, the only other predicate in the file was `skos:broadMatch`. I think that if we don't filter everything but `skos:exactMatch`, then we would need to somehow include the predicate also in the robot template, but IDK what would need to be done in that situation. My guess is it should only be exact match, but will see if Nico confirms.

Reply:
It is correct, they should be filtered by exact match. We need to be a bit conservative assigning the ORDO subsets - only to those classes that are identical with the Mondo classes!

Reply:
Gotcha, thanks for that requirement!